tests/resources/nimbus-testbeds/vic-nfs.rb

```diff
@@ -0,0 +1,58 @@
+#This file is needed because there are 3 functions in it which are used in this testbed.
+require '/mts/git/nimbus/lib/testframeworks/testng/testng.rb'
+oneGB = 1 * 1000 * 1000
+$testbed = Proc.new do |sharedStorageStyle, esxStyle|
+  esxStyle ||= 'fullInstall'
+  esxStyle = esxStyle.to_s
+  sharedStorageStyle = sharedStorageStyle.to_s
+
+  testbed = {
+    'version' => 3,
+    'name' => "vsan-1esx-allflash-#{sharedStorageStyle}-#{esxStyle}",
+    'esx' => (0..0).map do |i|
+      {
+        'name' => "esx.#{i}",
+        'style' => esxStyle,
+        'numMem' => 64 * 1024,
+        'numCPUs' => 6,
+        'disks' => [ 250 * oneGB ],
+        'disableNfsMounts' => true,
+        'ssds' => [ 200 * oneGB, 250 * oneGB ],
+        'nics' => 2,
+        'staf' => false,
+        'desiredPassword' => 'ca$hc0w',
+        'vmotionNics' => ['vmk0'],
+        'post_postboot_commands' => [
+            'esxcli system settings kernel set -s autoPartition -v "TRUE"', \
+            'esxcli system settings kernel set -s skipPartitioningSsds -v "TRUE"', \
+            'services.sh restart', \
+            'esxcli vsan storage tag add -d mpx.vmhba1:C0:T1:L0 -t capacityFlash', \
+            'esxcli storage core adapter rescan --all', \
+            'esxcli network firewall set --enabled false', \
+            'echo \'vhv.enable = "TRUE"\' >> /etc/vmware/config'
+            ],
+      }
+    end,
+    'vsan' => true,
+    'nfs' => [{
+    'name' => 'dev-nfs',
+    'type' => 'NFS',
+    'nfsOpt' => 'rw',
+    'numMem' => 64 * 1024,
+    },
+    'nfs' => {
+    'name' => 'dev-nfs',
+    'type' => 'NFS',
+    'nfsOpt' => 'ro',
+    'numMem' => 64 * 1024,
```

Comment:
I like the memory increase, might be overkill but is there any way to specify SSD for these?  and we should specify a good sized disk capacity in addition to the mem update